apps/web-giddh/src/app/expenses/components/rejected-list/rejected-list.component.scss

```diff
@@ -49,3 +49,12 @@
 .icon-pointer {
     cursor: pointer;
 }
+.mat-column-s_no {
```

Comment:
change this classname and add proper class, according to our guideline @ravinderwalkover

Reply:
@aditi-tiwari17 this is dynamic class, can't control it.